### ISSUE TITLE
Upgrading to Nova 3.21 breaks ConditionalContainer with HasMany field

### ISSUE THREAD

**Issue by kukukk** [dcasia/conditional-container#55]
After upgrading Nova to 3.21 (or 3.22) ConditionalContainer no longer works with HasMany field. It shows the loading icon, but in the background there is a GET request which returns 409 Conflict.

With Nova 3.20.0 it works.

**Comment by milewski** [comment]
Workaround:

```php
    public function fieldsForIndex(NovaRequest $request): array
    {
        return [
            HasMany::make('Your Has Many Resource'),
        ];
    }
```

**Comment by kukukk** [comment]
It works. Thanks!

**Comment by milewski** [comment]
I found a fix:

```php
public function hasRelatableField(NovaRequest $request, $attribute)
{
    return $this->availableFields($request)->whereInstanceOf(\Laravel\Nova\Contracts\RelatableField::class);
}
```

Would you be able to send a PR for this?

**Comment by kukukk** [comment]
Should it be a PR for Nova? I don't know where the Nova development is done.

Btw, it seems to be a bigger issue because it also break https://novapackages.com/packages/epartment/nova-dependency-container

**Comment by milewski** [comment]
Never mind I fixed it pushed on version [v1.3.0](https://github.com/dcasia/conditional-container/releases/tag/v1.3.0)
